mailpoet/lib/Util/Notices/DatabaseEngineNotice.php

```diff
@@ -54,12 +67,12 @@ private function checkTableEngines(): array {
       fn($row) => $row['Name'],
       array_filter(
         $data,
-        fn($row) => strtolower($row['Engine']) !== 'innodb'
+        fn($row) => is_string($row['Engine']) && (strtolower($row['Engine']) !== 'innodb')
```

Comment:
I don't know what to do if the key is not defined. Because if the query is misbehaving, a warning could be a good option. However, I added a check and if the key is not defined the user will not see the warning.